### ISSUE TITLE
No mention of privacy

### ISSUE THREAD

**Issue by joeyfortuna** [run-llama/llama_parse#188]
Please add documentation about how calls to the API are treated.
Are they stored and used for training?
Are they otherwise saved?
This service looks very promising but without that clarification, we can't endorse using it for business.

Thanks!

**Comment by lukebelbina** [comment]
Agreed, I'd like to use this for a business use case but the documents are not to be shared or used for training

**Comment by spreeni** [comment]
Also further transparency about which third-party services (and in which regions) are used would be nice.

**Comment by marco-bertelli** [comment]
Same Situation Here, the tool is awesome but unless we have GDPR info i can't use it with my team for production EU projects

**Comment by neuralnoy** [comment]
Have the same issue here. Is there any news on this?

**Comment by jdeepwell** [comment]
When I read about the cooperation with Microsoft/Azure

> [Announcing the LlamaIndex integration with Azure AI](https://www.llamaindex.ai/blog/announcing-the-llamaindex-integration-with-azure-ai?utm_source=drip&utm_medium=email&utm_campaign=LlamaIndex+Newsletter+2024-11-26)


I was hoping that there might be a way to run LlamaParse on Azure instances in the EU – but it does not seam so…

As long as there is no clear declaration in regards to GDPR, LlamaParse is basically unusable in Europe 😥
